tasks/flavors.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Ensure nova flavors exist
-  os_nova_flavor:
+  openstack.cloud.compute_flavor:
```

Comment:
It's been in the openstack.cloud ansible collection since the collection's version 1.0.0. `os_nova_flavor` was renamed to this just before the first version was tagged ([here](https://opendev.org/openstack/ansible-collections-openstack/commit/e47c4671c7f91bea630666584620faf65eab4ccb)). I'm not sure where to find which version of Ansible this correlates to

Reply:
> This redirect also works with Ansible 2.9.

So perhaps that's the earliest version that works? Probably we're safe to proceed on that basis.

Reply:
Should update `min_ansible_version` in `meta/main.yml`